cmake/features.cmake

```diff
@@ -112,7 +112,11 @@ ie_dependent_option(ENABLE_TBB_RELEASE_ONLY "Only Release TBB libraries are link
 
 ie_option (ENABLE_SYSTEM_PUGIXML "use the system copy of pugixml" OFF)
 
-ie_option (ENABLE_CPU_DEBUG_CAPS "enable CPU debug capabilities at runtime" OFF)
+ie_option (ENABLE_DEBUG_CAPS "enable OpeNVINO debug capabilities at runtime" OFF)
+
+ie_dependent_option (ENABLE_GPU_DEBUG_CAPS "enable GPU debug capabilities at runtime" ON "ENABLE_DEBUG_CAPS" OFF)
+
+ie_dependent_option (ENABLE_CPU_DEBUG_CAPS "enable CPU debug capabilities at runtime" ON "ENABLE_DEBUG_CAPS" OFF)
```

Comment:
I tried to turn on `ENABLE_GPU_DEBUG_CAPS` without turning on `ENABLE_DEBUG_CAPS` as below, and the behavior was somewhat confusing as CmakeCache.txt says it was `ON`, but it was actually `OFF`. Is it intended?
`cmake -DENABLE_GPU_DEBUG_CAPS=ON ..`

Reply:
This is expected behavior of [cmake dependent options](https://cmake.org/cmake/help/latest/module/CMakeDependentOption.html). If you don't enable `ENABLE_DEBUG_CAPS` option, then `ENABLE_CPU/GPU_DEBUG_CAPS` options are hidden from user. Any value that you set for these options is cached, but it will be used only when  `ENABLE_DEBUG_CAPS=ON`

Reply:
> Oh I see.. but isn't it too complicated to have two-stages? 

You don't need to specify both.

`-DENABLE_DEBUG_CAPS=ON` option automatically enables all dependent options. Component specific options are mostly needed to be able to *disable* debug caps for some components.